tests/by-util/test_ls.rs

```diff
@@ -4380,22 +4380,22 @@ fn test_tabsize_formatting() {
 
     scene
         .ucmd()
-        .args(&["-x", "-w18", "-T4"])
+        .args(&["-C", "-w18", "-T4"])
```

Comment:
Maybe we should test both `-x` and `-C`? The logic is the same in uutils-term-grid I guess, but we shouldn't assume too much about that here.